convert/src/lib.rs

```diff
@@ -0,0 +1,27 @@
+//! Convertion utilities between protobuf and mongodb
```

Comment:
should tests not go inside the tests.rs file?

Reply:
It’s the same thing, in theory it should be done to separate the tests from the rest

Reply:
Then I still think it should go inside tests.rs